packages/blockchain-wallet-v4-frontend/src/scenes/VerifyEmailToken/index.js

```diff
@@ -27,12 +30,21 @@ class VerifyEmailToken extends React.PureComponent {
     const isProdEnv = this.props.appEnv === 'prod'
 
     const link = isProdEnv
-      ? 'https://blockchain.page.link/email_verified'
-      : 'https://blockchainwalletstaging.page.link/email_verified'
+      ? 'https://blockchain.page.link/'
+      : 'https://blockchainwalletstaging.page.link/'
 
-    const isValidContext = VALID_CONTEXTS.indexOf(context) > -1
+    const params = new URLSearchParams()
+    params.set('deep_link_path', PARAM_DEEP_LINK_PATH)
+    params.set('isi', PARAM_ISI)
+    params.set('ibi', PARAM_IBI)
```

Comment:
One more param that needs to be added as well:

```
apn=piuk.blockchain.android
```

Reply:
Added, thanks!